admin/workspace-management/cvms/index.md

```diff
@@ -28,6 +28,20 @@ deployment option.
 > [Google](../../../setup/kubernetes/google.md). Azure-hosted clusters will meet
 > these requirements as long as you use Kubernetes version 1.18+.
 
+### HostPath mounts
+
+The host paths required for CVM functionality depend on whether **Caching** is
+enabled and on whether **Auto loading of the `shiftfs` kernel module`** is
+enabled. These settings can be found under **Manage > Admin > Infrastructure**.
+
+The below table documents the host paths that are mounted:
+
+| Caching | Off | On | On | | Auto load `shiftfs` | N/A | Off | On |
+|--------------------:|-----------|------------|------------| | `/usr/src` |
```

Comment:
something seems to have gotten mixed up during formatting?

Reply:
thanks, markdownlint!